hack/community/readme.md

```diff
@@ -1,45 +1,24 @@
-necessary fields such as the name, description, and image. It also generates a 
-signed commit ready to be pushed to community-operators-prod repo. 
+The generate.sh script is triggered to set up for the aws-e2e-operator-test 
+in all community-4.10+ branch PRs. The script copies over the WMCO community 
+bundle to the artifact directory while also replacing csv fields such as the 
+name, description, and date. Once generated into the artifact directory, the 
+bundle is used in the windows-e2e-operator test. After it passes, the bundle can
+be used to release a new community WMCO.
 
-## Pre-requisites
-Before running generate.sh, please ensure the following has merged:
-- openshift/release PR to add CI jobs to relevant community branch.
-[Example](https://github.com/openshift/release/pull/27081)
-- Submodule update PR for the relevant community branch. 
-- Ensure WMCB submodule is pointing to relevant community branch.
-[Example](https://github.com/openshift/windows-machine-config-operator/blob/community-4.10/.gitmodules#L4)
-- Fork and clone [community-operators-prod repo](https://github.com/redhat-openshift-ecosystem/community-operators-prod) 
-- Install yq:
-```shell script
-  curl -L https://github.com/mikefarah/yq/releases/download/v4.13.5/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
-```
-
-### From WMCO root, run:
-Parameter 3 can be grabbed from our [Quay repo](https://quay.io/repository/openshift-windows/community-windows-machine-config-operator?tab=tags).
-```shell script
-bash ./hack/community/generate.sh <community_wmco_version> <community_ocp_version> <latest_community-4.x_tag-commit-hash> <path/to/community-operators-prod>
-```
-Example: Cut community release v5.1.0 for community-4.10
-```shell script
-bash ./hack/community/generate.sh 5.1.0 community-4.10 community-4.10-hash ../community-operators-prod
-```
-
-### Review and test the operator
-- Scan over the newly generated CSV to ensure all the necessary fields were replaced
-- Follow: https://github.com/openshift/windows-machine-config-operator/blob/master/docs/HACKING.md#i-want-to-try-the-unreleased-wmco-and-deploy-it-mimicking-the-official-deployment-method 
-  You may skip step: Building the bundle image.
-- [CSV validation](https://operatorhub.io/preview) - paste in the CSV contents to
-preview what the community operator will look like in Operator Hub.
+### After the test passes
+- Copy the community bundle from the ARTIFACT_DIR.
+- Scan over the bundle CSV and annotations to ensure all the necessary fields 
+were replaced.
 
+### Open a PR
+- Fork and clone [community-operators-prod](https://github.com/redhat-openshift-ecosystem/community-operators-prod)
+repo.
+- Paste the bundle contents into a new folder under 
+community-windows-machine-config-operator. The folder title should be the
+next version of the community WMCO that is to be released.
+- A few fields in the CSV may need to be added or adjusted including
+olm.skipRange, name, version, and replaces-mode.
```

Comment:
Can't this be scripted too? i.e. give the generate script the location of the community-operators-prod on disk and copy over what is needed and adjust the fields in the CSV. The only thing that can be manual is opening the PR.

Reply:
PS: this can be a follow up story / PR.

Reply:
It used to be scripted. I'm removing those parts in this PR. Some of the bundle changes required for the community-operators-prod repo don't work well with the windows-e2e-operator-test. I agree it should be a follow up story / PR.